src/utils/getThemeValue.js

```diff
@@ -21,4 +21,4 @@ export default (...args) => ({
       throw new ReferenceError("value is not defined");
     }
     return acc[el];
-  }, themes[themeName]);
+  }, themes[themeName] || { ...themes.global, ...themeName });
```

Comment:
It seems strange to index the theme object under the `themeName` key... what do you think about passing the custom `theme` object indexed under the `theme` key itself? I think either way we should validate that the custom `theme` is an object before spreading its contents.